angular-tree-control.js

```diff
@@ -269,6 +317,15 @@
                         element.empty();
                         element.append(clone);
                     });
+
+                    var parentIndex = scope.$parent.visibleNodes.indexOf(scope.$parent.node);
+                    if(parentIndex !== -1) {
+                        var myIndex = 1 + scope.transcludeScope.$parentNode.children.indexOf(scope.node);
```

Comment:
should change to:
var myIndex = 1 + scope.transcludeScope.$parentNode[scope.$parent.options.nodeChildren].indexOf(scope.node);
to keep the custom children node feature working